tango_ros_common/tango_ros_messages/srv/SaveMap.srv

```diff
@@ -1,6 +1,11 @@
+int8 LOC_AND_NAV_MAPS=0
+int8 ONLY_LOC_MAP=1
+int8 ONLY_NAV_MAP=2
+int8 request
 string map_name
 ---
 string message
-string map_name
-string map_uuid
+string localization_map_name
+string localization_map_uuid
+string navigation_map_name
```

Comment:
Will the occupancy grid map always be used for navigation or could it also be used to e.g. represent the walls in a game environment or to measure the surface of the walls in a home? In this case, I would rename all occurences to occupancy_grid.